tests/framework/ROM/TimeSeries/ARMA/clustered_choice_rand.xml

```diff
@@ -1,11 +1,12 @@
+<?xml version="1.0" ?>
 <Simulation verbosity="debug">
   <TestInfo>
     <name>framework/ROM/TimeSeries/ARMA.ClusteredRandChoice</name>
     <author>alfoa</author>
     <created>2020-04-15</created>
     <classesTested>SupervisedLearning.ARMA</classesTested>
     <description>
-      Tests the mechanics of clusterering the "time" subspace of the ARMA ROM, with
+      Tests the mechanics of clusterering the &quot;time&quot; subspace of the ARMA ROM, with
```

Comment:
change &quot to ``''?

Reply:
this is an interesting effect of going through the conversion script that should probably be investigated. I will change these.